resources/es/cemu.po

```diff
@@ -1855,59 +1843,59 @@ msgstr "&Taiwán"
 
 #: MainWindow.cpp
 msgid "&English"
-msgstr "&Inglés"
+msgstr "&inglés"
 
 #: MainWindow.cpp
 msgid "&Japanese"
-msgstr "&Japonés"
+msgstr "&japonés"
 
 #: MainWindow.cpp
 msgid "&French"
-msgstr "&Francés"
+msgstr "&francés"
 
 #: MainWindow.cpp
 msgid "&German"
-msgstr "&Alemán"
+msgstr "&alemán"
 
 #: MainWindow.cpp
 msgid "&Italian"
-msgstr "&Italiano"
+msgstr "&italiano"
 
 #: MainWindow.cpp
 msgid "&Spanish"
-msgstr "&Español"
+msgstr "&español"
 
 #: MainWindow.cpp
 msgid "&Chinese"
-msgstr "&Chino"
+msgstr "&chino"
 
 #: MainWindow.cpp
 msgid "&Korean"
-msgstr "&Coreano"
+msgstr "&coreano"
 
 #: MainWindow.cpp
 msgid "&Dutch"
-msgstr "&Neerlandés"
+msgstr "&neerlandés"
 
 #: MainWindow.cpp
 msgid "&Portuguese"
-msgstr "&Portugués"
+msgstr "&portugués"
 
 #: MainWindow.cpp
 msgid "&Russian"
-msgstr "&Ruso"
+msgstr "&ruso"
 
 #: MainWindow.cpp
 msgid "&Taiwanese"
-msgstr "&Taiwanés"
+msgstr "&taiwanés"
```

Comment:
I'm not sure whether these changes should be done. These strings are used in a menu, and menu entries always start with a capital letter.

Reply:
In Spanish languages names are common nouns, they SHOULD never be capitalised (as names)

Reply:
In Italian language names are common nouns, too. But the fact that they are capitalized because they're at the beginning of a sentence doesn't really break that rule IMHO.

Anyway, if you deem these strings to be more correct as they are, I won't force you to change them.